### PR TITLE
Add test environment

### PR DESCRIPTION
**Pull request by alexander-schranz**
Add test environment to avoid common pitfalls in incorrect configuration for security setup.

You can use the following `patch` file to have some basic tests:

https://gist.githubusercontent.com/alexander-schranz/8a0dda598537ad5740faefe6333637e8/raw/91a7ca6fc534fd6cacab2eead8a549e1f3a2b07d/test-case.patch

**To be discussed**:

 - Should we also add a abstract `AdminTestCase` and `WebsiteTestCase` so its easier to create functional tests foreach Kernel (See [patch](https://gist.githubusercontent.com/alexander-schranz/8a0dda598537ad5740faefe6333637e8/raw/91a7ca6fc534fd6cacab2eead8a549e1f3a2b07d/test-case.patch) file for example).

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -14,6 +14,10 @@
             "app/WebsiteKernel.php"
         ]
     },
+    "autoload-dev": {
+        "psr-4": { "Tests\\": "tests/" },
+        "files": [ "vendor/symfony/symfony/src/Symfony/Component/VarDumper/Resources/functions/dump.php" ]
```
Comment (danrot) [inline composer.json]: Why do we need this?

File: phpunit.xml.dist [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+            <exclude>
+                <directory>src/*Bundle/Resources</directory>
+                <directory>src/*/*Bundle/Resources</directory>
+                <directory>src/*/Bundle/*Bundle/Resources</directory>
+            </exclude>
+        </whitelist>
+    </filter>
+
+    <listeners>
+        <listener class="Symfony\Bridge\PhpUnit\SymfonyTestsListener" />
```
Comment (danrot) [inline phpunit.xml.dist]: Is that really needed? It's also not in symfony-standard...

File: composer.json (hunk 1 above)
Comment (alexander-schranz) [reply composer.json]: not sure why its in [symfony-standard](https://github.com/symfony/symfony-standard/blob/3.3/composer.json#L14) did try to use the dump function did work without problems in the tests. so will remove it.

File: phpunit.xml.dist (hunk 2 above)
Comment (alexander-schranz) [reply phpunit.xml.dist]: seems not been needed when using `bin/simple-php-unit` from [symfony](https://symfony.com/doc/current/components/phpunit_bridge.html):

**Pull request merged**